_data/backup.yml

```
@@ -141,6 +141,14 @@ websites:
     doc: https://support.microsoft.com/en-us/help/12408/
     exception: "2FA phone calls may not be available in all regions"
 
+  - name: OneHub
+    url: https://www.onehub.com/
+    img: onehub.svg
+    tfa:
+      - sms
+      - totp
+    doc: https://support.onehub.com/hc/en-us/articles/360039792571
+
```

```suggestion
    exception: "Software 2FA requires adding a phone number as a fallback."
    
```
According to the doc page:
> A mobile phone number is required as a backup, in case you lose access to your authenticator.

Perhaps an exception should be added?